scripts/mock_llm.py

```
@@ -0,0 +1,48 @@
+                "usage": {"prompt_tokens":1,"completion_tokens":1,"total_tokens":2}
+            }
+            self._send(200, resp)
+        else:
+            self._send(404, {"error":"not found"})
+
+if __name__ == '__main__':
+    port = 11434
+    if len(sys.argv) > 1:
+        try:
+            port = int(sys.argv[1])
+        except:
```

Ruff (0.15.20)

[error] Do not use bare `except`

(E722)